truss/templates/server/model_wrapper.py

```diff
@@ -321,10 +322,11 @@ def from_model(cls, model_cls) -> "ModelDescriptor":
             truss_schema = cls._gen_truss_schema(
                 predict=predict, preprocess=preprocess, postprocess=postprocess
             )
-
         else:
+            # This case should never happen, since above conditions should
+            # be exhaustive.
             raise errors.ModelDefinitionError(
-                f"Truss model must have a `{MethodName.PREDICT}` or `{MethodName.WEBSOCKET}` method."
+                "Unsupported method combination on truss model."
```

Comment:
This doesn't quite feel like a ModelDefinitionError anymore right? If we expect the above conditions to be exhaustive

I'm not sure if we have any other generic error that suggests an internal framework problem instead? No strong preference though